spec/factories/user.rb

```diff
@@ -6,5 +6,10 @@
     password              { Faker::Internet.password }
     first_name            { Faker::Name.first_name }
     last_name             { Faker::Name.last_name }
+    locale                { 'en' }
+  end
+
+  trait :lang_es do
+    locale :es
```

Comment:
`locale { 'es' }` Try to keep the data type consistant

Reply:
Actually, it's okay to use it here.

Check this post http://arjanvandergaag.nl/blog/factory_girl_tips.html

Common pitfalls when creating factories -> 3. Do not use dynamic values by default